build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.5
+FROM golang:1.5.1
```

Comment:
is this actually required? i think 1.5 refers to 1.5.1 now (and generally floats)

probably just need to fetch golang:1.5